src/splitInteger.test.js

```diff
@@ -3,19 +3,20 @@
 describe(`Function 'splitInteger':`, () => {
   const splitInteger = require('./splitInteger');
 
-  it(`function 'splitInteger' should be declared`, () => {
+  it(`should be declared`, () => {
     expect(splitInteger).toBeInstanceOf(Function);
   });
 
   it(`should return an array`, () => {
     expect(splitInteger()).toBeInstanceOf(Array);
   });
 
-  it(`variables should be of type 'number'`, () => {
+  it(`values should be of type 'number'`, () => {
     expect(splitInteger()).not.toBeNaN();
   });
```

Comment:
Here you check that the result of the function is not `NaN`. What for? If you want, you can check that each element in the array is a number, try to figure out how to do it.